service_analysis/app/controllers/AnalysisController.scala

```diff
@@ -0,0 +1,129 @@
+package controllers
+
+import com.google.inject.{Inject, Singleton}
+import models.events.AnalysisResults.AnalysisResult
+import models.events.SaveCommands._
+import models.events._
+import no.uio.musit.MusitResults.{MusitError, MusitResult, MusitSuccess}
+import no.uio.musit.models.{CollectionUUID, EventId, ObjectUUID}
+import no.uio.musit.security.Authenticator
+import no.uio.musit.service.MusitController
+import play.api.Logger
+import play.api.libs.concurrent.Execution.Implicits.defaultContext
+import play.api.libs.json._
+import play.api.mvc.Result
+import services.AnalysisService
+
+import scala.concurrent.Future
+
+@Singleton
+class AnalysisController @Inject() (
+    val authService: Authenticator,
+    val analysisService: AnalysisService
+) extends MusitController {
+
+  val logger = Logger(classOf[AnalysisController])
+
+  lazy val internalErr = (msg: String) => InternalServerError(Json.obj("message" -> msg))
```

Comment:
We're only using this in context of `MusitError`, should we have changed the signature to a `MusitError`? It will make it less generic.